src/App.jsx

```diff
@@ -1,104 +1,110 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+import { PostList } from './components/PostList';
```

Comment:
Ensure that the `PostList` component is properly defined and exported in the `./components/PostList` file. The current code imports `PostList`, but it is commented out in the code below, which will cause an error when rendering the `App` component.